charts/kyverno/values.yaml

```diff
@@ -1,6 +1,8 @@
 nameOverride:
 fullnameOverride:
 namespace:
+# Supported- default/restricted
+policyType: default
```

Comment:
can we rename this to `podSecurityPolicies`?

Reply:
You mean `podSecurityStandards` as mentioned here- https://github.com/kyverno/kyverno/pull/1493#issuecomment-766295662